docs/getting-started/wallets.md

```diff
@@ -1,289 +1,102 @@
-
-### Mnemonic
-
-:::danger Always keep your mnemonic safe 
-Anyone who knows the mnemonic for your wallet account can access your TAO tokens. Hence, you must always keep this mnemonic in a safe and secure place known only to you. More importantly, if you lose your wallet address, you can use its mnemonic (stored away in safekeeping) to restore the wallet.
-:::
-
-:::note Use Import option in Chrome Wallet Extension
-To restore your lost coldkey, use the **Import** option in Chrome Extension for Bittensor Wallet and provide your 12-word mnemonic.
-:::
-
-:::tip Suitable for subnet participation
-Use this option if you are creating a subnet or participating as a subnet validator or a subnet miner. You must [install Bittensor](installation.md) for this option.
-:::
-
-After you have [installed Bittensor](installation.md), you can create a local wallet on your machine in the following two ways:
-
-- [Using `btcli`](#creating-a-local-wallet-with-cli)
-- [Using Python](#creating-a-local-wallet-using-python)
-
-## Creating a local wallet with CLI
-
-### Coldkey and hotkey
-
-A Bittensor wallet consists of a **coldkey** and a **hotkey**. Only coldkey is created when you use the [Chrome Extension for Bittensor Wallet](https://chromewebstore.google.com/detail/bittensor-wallet/bdgmdoedahdcjmpmifafdhnffjinddgc?pli=1). This is sufficient for normal storage and sending and receiving of TAO. However, to participate in a subnet, you will need a local coldkey and a local hotkey.
-
-:::tip Explanation of keys 
-See [Working with Keys](../subnets/working-with-keys.md) for an explanation of coldkey and hotkey.
+:::tip Coldkey and hotkey each are pairings of separate private and public keys
+Each key is a pairing of two separate [EdDSA cryptographic keypairs](https://en.wikipedia.org/wiki/EdDSA#Ed25519). Hence, a coldkey is a pairing of a private key and a public key. Similarly, a hotkey is a pairing of another set of private key and public key. In this sense, a coldkey or a hotkey is each analogous to an account on a blockchain, where the account is defined by a pair of a public and a private key.
 :::
 
-### Creating a coldkey using `btcli` 
-
-If you plan to perform any of the following tasks, you only need to create a coldkey:
-
-- Create a subnet.
-- Transfer TAO.
-- Delegate to a validator-delegate's hotkey.
+### Coldkey
 
-**However, if you want to validate or mine in a subnet, you will need to create hotkey also**. See the below section [Creating a hotkey using `btcli`](#creating-a-hotkey-using-btcli).
+The coldkey is synonymous with the wallet name. For example, the `--wallet.name` option in a `btcli` command always accepts only `<coldkey>` as its value and the `--wallet.hotkey` option only accepts `<hotkey>` as its value. This is because one coldkey can have multiple hotkeys; hence, the wallet name is assigned to the coldkey.
```

Comment:
```suggestion
In `btcli`, the coldkey is synonymous with the wallet name. For example, the `--wallet.name` option in a `btcli` command always accepts only `<coldkey>` as its value and the `--wallet.hotkey` option only accepts `<hotkey>` as its value. This is because the coldkey holds the permissions and ownership over multiple hotkeys on-chain; hence, the wallet name is assigned to the coldkey.
```

Reply:
Importantly, the coldkey is called "cold" because it is meant to be kept *more secure* (cold) and used less often. Rather than a hotkey which is "hot" and used to sign messages frequently, requiring it be unencrypted in memory and/or storage. Whereas the coldkey is kept encrypted in storage only to be decrypted shortly for use by the user directly.